scripts/config_test_vm.psm1

```diff
@@ -210,6 +211,52 @@ function Export-BuildArtifactsToVMs
     Remove-Item -Force $tempFileName
 }
 
+#
+# Install eBPF components on VM.
+#
+
+function Install-eBPFComponentsOnVM
+{
+    param([parameter(Mandatory=$true)][string] $VMName,
+          [parameter(Mandatory=$true)][bool] $KmTracing,
+          [parameter(Mandatory=$true)][string] $KmTraceType)
+
+    Write-Log "Installing eBPF components on $VMName"
+    $TestCredential = New-Credential -Username $Admin -AdminPassword $AdminPassword
+
+    Invoke-Command -VMName $VMName -Credential $TestCredential -ScriptBlock {
+        param([Parameter(Mandatory=$True)] [string] $WorkingDirectory,
+              [Parameter(Mandatory=$True)] [string] $LogFileName,
+              [Parameter(Mandatory=$true)] [bool] $KmTracing,
+              [Parameter(Mandatory=$true)] [string] $KmTraceType)
+        $WorkingDirectory = "$env:SystemDrive\$WorkingDirectory"
+        Import-Module $WorkingDirectory\common.psm1 -ArgumentList ($LogFileName) -Force -WarningAction SilentlyContinue
+        Import-Module $WorkingDirectory\install_ebpf.psm1 -ArgumentList ($WorkingDirectory, $LogFileName) -Force -WarningAction SilentlyContinue
+
+        Install-eBPFComponents -KmTracing $KmTracing -KmTraceType $KmTraceType -KMDFVerifier $true -ErrorAction Stop
```

Comment:
nit: In the previous version, there was also a call to `Enable-KMDFVerifier`, but that is removed now. Is that expected?

Reply:
No it's not removed, just moved into `install_ebpf.ps1` so to be more contextually close to tracing enablement.